source/addonHandler/__init__.py

```diff
@@ -602,6 +603,55 @@ def _translatedManifestPaths(lang=None, forBundle=False):
 	return [sep.join(("locale", lang, MANIFEST_FILENAME)) for lang in langs]
 
 
+def initializeModulePackagePaths():

```

Comment:
rather than making `__init__.py` larger, would it make sense to package these new functions in `addonHandler/packaging.py` or something similar?

Reply:
Good idea, I"ll do that